Add News page and some initial posts (#18)
djangocon/2023.djangocon.eu: _posts/2023-01-01-announcing-djangocon-europe-2023.md

---
author: DjangoCon Europe Organisers
category: General
date: 2023-01-01 06:00:00
layout: post
published: true
post_photo_alt: The view of Edinburgh from Calton Hill
post_photo_url: /static/img/locations/calton-hill.jpg
title: DjangoCon Europe is coming to Edinburgh!
---

Hello, everyone!  This year, DjangoCon Europe is coming to Edinburgh, Scotland!

We are very excited to announce that the website for DjangoCon Europe 2023 is [live](https://2023.djangocon.eu).

> 
> Many thanks to [TorchBox](https://torchbox.com/) for the website design!

Reviewer: ```suggestion

Many thanks to [Torchbox](https://torchbox.com/) for the website design!
```

The conference will be from Monday 29th May to Friday 2nd June 2023.

- 29-31 May: Talks, Tutorials and Workshops
- 1-2 June: Sprints

Talks, tutorials and workshops will be held at the [Assembly Rooms](/venues/#talks), in 
the heart of Edinburgh's city centre. Sprints will be held at the 

> [Biscuit Factory](/venues/#sprints) in Leith.

Reviewer: I see we have an issue with the site stylesheet adapting to those anchor links. Will change the CSS.

We'll also have a social event at an amazing venue, [The Caves](/venues/#social)!

> The Call for Proposals and ticket sales will be open soon.  In the meantime, you can get an

Reviewer: ```suggestion
The Call for Proposals and ticket sales will be open soon. In the meantime, you can get an
```

> insight into the city [here](/location). 

Reviewer: ```suggestion
insight into the city on our [Location page](/location). 
```

> We hope we'll see you all at DjangoCon Europe 2023, and don't forget to follow us [@DjangoConEurope](https://twitter.com/djangoconeurope) on Twitter

Reviewer: ```suggestion
We hope we'll see you all at DjangoCon Europe 2023, and don't forget to follow us [@{{ site.twitter.username }}](https://twitter.com/{{ site.twitter.username }}) on Twitter.
```
